benefits/static/css/styles.css

```diff
@@ -1,6 +1,6 @@
 :root {
-  --primary-color: #046b99;
-  --primary-color-rgb: 4, 107, 153;
+  --primary-color: #045b86;
+  --primary-color-rgb: 4, 91, 134;
```

Comment:
I thought we wanted to transition to using RGB everywhere since it is easier to diff and get the alpha correct?

It seems like this variable is only used to define (overwrite) the `--bs-primary-rgb` variable, whereas our `--primary-color` variable is used extensively.

Reply:
`rgb` is used by Bootstrap for specific areas that need a rgbA. we could refactor later to get rid of the hex numbers entirely. it would require us to change the variable declaration a little bit though (something to make sure the variable is wrapped in`rgb()`).

Reply:
I'm following Bootstrap CSS Variable conventions: https://getbootstrap.com/docs/5.2/customize/css-variables/#root-variables by declaring both.